src/report.tex

```diff
@@ -202,7 +207,16 @@ \section*{Executive Summary}
 \\
 \\
 This analysis focused on only shutdown photon sources, and therefore does not
-contain prompt photon doses.
+contain prompt photon doses which are not present following machine shutdown.
+\\
+\\
+Future studies should focus on further decreasing the SDR in the person 
+access required areas such as the equatorial and upper port interspaces. A 
+significant finding in the upper ports was that a significant fraction of the
+SDR comes from decay photons created outside the upper port, from the activation
+of the structural components nearby. Focus should be placed on improving 
+shielding internal to the strong sources, possible on improving the \gls{vv} filler
+near the \gls{pf}2 coil or strong local shielding. 
```

Comment:
This acronym won't expand well, I don't think...  maybe "\gls{pf} \#2" or "\gls{pf} coil \#2"?